services/apps/integration_run_worker/src/queue/index.ts

```
@@ -70,11 +70,17 @@ export class WorkerQueueReceiver extends PrioritizedQueueReciever {
             msg.onboarding,
             msg.isManualRun,
             msg.manualSettings,
+            msg.additionalInfo,
           )
           break
         case IntegrationRunWorkerQueueMessageType.GENERATE_RUN_STREAMS:
           const msg2 = message as GenerateRunStreamsQueueMessage
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)